### ISSUE TITLE
documentation for updating the R package?

### ISSUE THREAD

**Issue by letitbk** [BrianAronson/docshop#3]
Brian, are there some useful resources that people can read and study when they want to update this package (they = I for now, but later probably other people might also want to do the same thing)? If so, could you add some documentation to the wiki page?

**Comment by BrianAronson** [comment]
Yes. Good point. I'll make some documentation and maybe even create a
script for simplifying the steps.

**Comment by BrianAronson** [comment]
Hi BK, I made some detailed tutorials in the wiki, and I created a couple of extra wiki pages that might want to fleshed out. Please see https://github.com/BrianAronson/docshop/wiki/Updating-the-Package

**Comment by letitbk** [comment]
This is awesome! Thanks a lot for doing this. I will try to edit our R package in some ways, and see what happens soon.
